test/unit/smartapp-initialize-spec.js

```diff
@@ -0,0 +1,130 @@
+const assert = require('assert').strict
+const SmartApp = require('../../lib/smart-app')
+
+describe('smartapp-initialize-spec', () => {
+	it('default handler', async () => {
+		const app = new SmartApp()
+			.appId('xxx')
+			.firstPageId('page1')
+			.permissions(['r:devices:*'])
+			.disableCustomDisplayName(true)
+			.disableRemoveApp(true)
+
+		// Initialize configuration callback
+		const resp = await app.handleMockCallback({
+			lifecycle: 'CONFIGURATION',
+			executionId: 'e6903fe6-f88f-da69-4c12-e2802606ccbc',
+			locale: 'en',
+			version: '0.1.0',
+			client: {
+				os: 'ios',
+				version: '0.0.0',
+				language: 'en-US'
+			},
+			configurationData: {
+				installedAppId: '7d7fa36d-0ad9-4893-985c-6b75858e38e4',
+				phase: 'INITIALIZE',
+				pageId: '',
+				previousPageId: '',
+				config: {}
+			},
+			settings: {}
+		})
+
+		const expectedInitResponse = {initialize: {
+			id: 'xxx',
+			firstPageId: 'page1',
+			permissions: ['r:devices:*'],
+			disableCustomDisplayName: true,
+			disableRemoveApp: true
+		}}
+
+		assert.deepStrictEqual(resp.configurationData, expectedInitResponse)
+	})
+
+	it('default handler', async () => {
```

Comment:
This has the same name as the test above, but does something different. It would be great to reflect the difference in the name somehow.